README.md

```diff
@@ -63,16 +63,11 @@ $ java -jar /tmp/swagger-codegen-cli.jar generate -l go -i gate-swagger.json -o
 
 ## How to run?
 
-This project uses the Go module system.
 First, make sure you have some version of Spinnaker running.
 
-Get the repository using `go get`:
-```bash
-go get github.com/codilime/floodgate
-```
-This will download, compile and install Floodgate to `$GOPATH/bin`. If this directory is present in your `$PATH`, the `floodgate` command will be available from any directory.
+Download proper binary from [releases page](https://github.com/codilime/floodgate/releases). Note, that you should download proper binary for your Spinnaker version ( gate part in binary name ).
```

Comment:
I would remove those additional spaces after and before round brackets - giving: `(gate part in binary name)`

Reply:
I would also split this line in half.